### PR TITLE
add Image and Input page

### PR DESCRIPTION
**Pull request by hsnice16**
implemented Image and Input page.
Live: https://poshui-documentation-image-input.netlify.app/pages/image.html

### CODE REVIEW

**Comment by BalajiHarinath** [comment]
When I resized the browser to see the effect of the responsive image, on scrolling the browser horizontally I noticed that your side bar is sticky and overlaps the documentation part. Please fix it.

**Comment by hsnice16** [comment]
Yeah, Balaji. The Library is not responsive yet, I will make it responsive after adding all the components.
Thanks for pointing this, and thankyou to review.

**Review by b0llu — APPROVED** [review]
Great choice of images! Clean code looks good to me 👍

File: pages/input.html [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+            </li>
+            <li class="d-i-block margin-lr-1">
+              <a class="link" href="introduction.html">Documentation</a>
+            </li>
+            <li class="d-i-block margin-lr-1">
+              <a class="link"
+                href="https://github.com/hsnice16/PoshUI-Documentation"
+                target="_blank"
+                >GitHub</a
+              >
```
Comment (vrx29) [inline pages/input.html:79]: just a typo i think. Please format these tags closing tag should be in previous line

Comment (hsnice16) [reply pages/input.html:79]: Actually, I tried to do this but prettier moves it onto the next line.

Comment (vrx29) [reply pages/input.html:79]: No issues everything else is good. You can go ahead.

**Pull request merged**